main.nf

```diff
@@ -406,8 +407,9 @@ workflow {
     picard_collectrnaseqmetrics(ch_indexed_bam, ch_refflat)
 
     // Assemble transcripts
-    stringtie(ch_indexed_bam, ch_gtf, ch_tab)
-    gffcompare(stringtie.out, ch_gtf)
+    stringtie(ch_indexed_bam, ch_gtf)
+    gffcompare(stringtie.out, ch_gtf,ch_tab)
```

Comment:
```suggestion
    gffcompare(stringtie.out.gtf, ch_gtf)
```

Reply:
We're not using the tab file in gff compare right?

Reply:
Currently the tab is not used anywhere =P.
But it is a very nice to have output!

Reply:
Indeed!